### ISSUE TITLE
FNX-3406 ⁃ [Bug]Common myth hyperlink and other links in app provide confusing description and action info for screen readers

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/fenix#10285]
Follow-up for #7887.

We have several links in app that are announced as a button would be.
This prevents screen readers users to have a complete and correct info regarding the element type and available action for element.

Proposed solution: 
<strike>Announce action as "_Follow link_" ("Double tap to follow link") and Textview type as "_Link_".</strike>
Edit : Should be announced as "_link_", without double tap and long press action.

**Comment by mcarare** [comment]
@asadotzler any thoughts on this issue or the proposed solution? TY!

**Comment by eeejay** [comment]
Hi! I don't know what gets posted where. But I added an implementation suggestion here:
https://github.com/mcarare/fenix/commit/eb0828c2e765804f0b7324bec99f3cc252463533#r38963440

**Comment by sflorean** [comment]
Verified as fixed on Nightly 7/24 and Beta 79.0.0 with Samsung Galaxy Note 10 (Android 10).
